examples/device/CMakeLists.txt

```diff
@@ -0,0 +1,27 @@
+cmake_minimum_required(VERSION 3.5)
+
+include(${CMAKE_CURRENT_SOURCE_DIR}/../../hw/bsp/${FAMILY}/family.cmake)
+
+project(tinyusb_device_examples)
```

Comment:
I have an issue running this CMakelits.txt file, it throws a warning when running cmake.  

```
$ examples/device/build$ cmake .. -DFAMILY=rp2040 -DBOARD=raspberry_pi_pico -DPICO_BUILD_DOCS=0
CMake Warning at /home/hathach/Documents/code/pico/pico-sdk/src/rp2_common/tinyusb/CMakeLists.txt:14 (message):
  PICO_TINYUSB_PATH specified but content not present.
```

when running make with generated project, it has several missing include

```
/home/hathach/Dropbox/tinyusb/tinyusb/examples/device/cdc_msc/src/msc_disk.c:26:10: fatal error: bsp/board.h: No such file or directory
   26 | #include "bsp/board.h"
      |          ^~~~~~~~~~~~~
compilation terminated.
make[2]: *** [cdc_msc/CMakeFiles/raspberry_pi_pico-cdc_msc.dir/build.make:76: cdc_msc/CMakeFiles/raspberry_pi_pico-cdc_msc.dir/src/msc_disk.c.obj] Error 1
make[2]: *** Waiting for unfinished jobs....
/home/hathach/Dropbox/tinyusb/tinyusb/examples/device/cdc_msc/src/usb_descriptors.c:26:10: fatal error: tusb.h: No such file or directory
   26 | #include "tusb.h"
      |          ^~~~~~~~
compilation terminated.
make[2]: *** [cdc_msc/CMakeFiles/raspberry_pi_pico-cdc_msc.dir/build.make:89: cdc_msc/CMakeFiles/raspberry_pi_pico-cdc_msc.dir/src/usb_descriptors.c.obj] Error 1
/home/hathach/Dropbox/tinyusb/tinyusb/examples/device/cdc_msc/src/main.c:30:10: fatal error: bsp/board.h: No such file or directory
   30 | #include "bsp/board.h"
      |          ^~~~~~~~~~~~~
compilation terminated.
make[2]: *** [cdc_msc/CMakeFiles/raspberry_pi_pico-cdc_msc.dir/build.make:63: cdc_msc/CMakeFiles/raspberry_pi_pico-cdc_msc.dir/src/main.c.obj] Error 1
make[1]: *** [CMakeFiles/Makefile2:2426: cdc_msc/CMakeFiles/raspberry_pi_pico-cdc_msc.dir/all] Error 2
make: *** [Makefile:84: all] Error 2
```

Reply:
should be fixed now